### PR TITLE
Add dependency list

### PR DESCRIPTION
**Pull request by curquiza**
Also, update the release guide.

### CODE REVIEW

File: resources/integration-release.md [hunk 1]
```diff
@@ -40,19 +40,25 @@ Also, when writing the final changelogs, you might notice some inconsistencies,
...
+The situation is as follows: `meilisearch-js@v0.2.0` is released on GitHub, the tag `v0.2.0` is created, but it is not published on [npm](https://www.npmjs.com/) which is still at `v0.1.0`.
 
-To solve the situation, I delete the release and the tag on GitHub. I merge the PR with the right version in `package.json`. I go back to the release screen, and I publish the newly created release draft. 
+To solve the situation, I delete the release and the tag on GitHub. I merge the PR with the right version in `package.json`. I go back to the release screen, and I publish the newly created release draft.
+
+## Release a beta version
+
+For some integrations, we need to release a beta version of the integration before releasing the official one. For instance, during the MeiliSearch pre-release, we often need to release a beta version of [meilisearch-js](https://github.com/meilisearch/meilisearch-js) to make the [mini-dashboard](https://github.com/meilisearch/mini-dashboard)) work with the futur release of MeiliSearch.
```
Comment (alallema) [inline resources/integration-release.md:62]:
```suggestion
For some integrations, we need to release a beta version of the integration before releasing the official one. For instance, during the MeiliSearch pre-release, we often need to release a beta version of [meilisearch-js](https://github.com/meilisearch/meilisearch-js) to make the [mini-dashboard](https://github.com/meilisearch/mini-dashboard)) work with the future release of MeiliSearch.
```

**Review by alallema — COMMENTED** [review]
I like this new guide!

File: resources/dependencies.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+⚠️ Also, even if they are not parts of the integration team, we should keep in mind the following projects strongly depend on `instant-meilisearch`:
+- the [`mini-dashboard`](https://github.com/meilisearch/mini-dashboard)
+- the demos of the DevRel team, like [the MoMA demo](https://github.com/meilisearch/demo-MoMA)
+
+## Docs-searchbar.js
+
+They depend on [`docs-searchbar.js`](https://github.com/meilisearch/docs-searchbar.js)
+- [`vuepress-plugin-meilsearch`](https://github.com/meilisearch/vuepress-plugin-meilisearch/)
+
```
Comment (bidoubiwa) [inline resources/dependencies.md:48]:
```suggestion

## Vuepress plugin

No dependencies for [vuepress-plugin-meilisearch]((https://github.com/meilisearch/vuepress-plugin-meilisearch/)

⚠️ Also, even if they are not parts of the integration team, we should keep in mind the following projects strongly depend on `vuepress-plugin-meilisearch`:

- The [documentation](https://docs.meilisearch.com/)

## Docs scraper

No dependencies for [docs-scraper](https://github.com/meilisearch/docs-scraper/).

⚠️ Also, even if they are not parts of the integration team, we should keep in mind the following projects strongly depend on `docs-scraper`:

- The [documentation](https://docs.meilisearch.com/)

```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

**Review by alallema — APPROVED** [review]
LGTM! 🍣

File: resources/dependencies.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+⚠️ Also, even if they are not parts of the integration team, we should keep in mind the following projects strongly depend on `meilisearch-ruby`:
+- the [`meilisearch-cloud`](https://github.com/meilisearch/meilisearch-cloud) backend (link only accessible internally)
+
+## Instant MeiliSearch
+
+They depend on [`instant-meilsearch`](https://github.com/meilisearch/instant-meilisearch/)
+- [`meilisearch-angular`](https://github.com/meilisearch/meilisearch-angular/) and the [associated code-sandbox](https://codesandbox.io/s/im-angularis-7xipe?file=/src/app/app.component.ts)
+- [`meilisearch-react`](https://github.com/meilisearch/meilisearch-react/) and the [associated code-sandbox](https://codesandbox.io/s/ms-react-is-sh9ud?fontsize=14&hidenavigation=1&theme=dark)
+- [`meilisearch-vue`](https://github.com/meilisearch/meilisearch-vue/) and the [associated code-sandbox](https://codesandbox.io/s/ms-vue-is-1d6bi?fontsize=14&hidenavigation=1&theme=dark)
+
```
[the remark this answers is not in the document]
Comment (curquiza) [reply resources/dependencies.md:39]: Yes, but not the scope of the integration team

File: resources/dependencies.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+# Dependencies between the repositories
+
+This resource is an exhaustive list of the existing dependencies between the integration team projects.
+
+## JavaScript SDK
+
+They depend on [`meilisearch-js`](https://github.com/meilisearch/meilisearch-js):
```
[the remark this answers is not in the document]
Comment (curquiza) [reply resources/dependencies.md:7]: They are not the dependencies of the JS SDK. They have the JS SDK as a dependency. So I would rather keep "they depend on the meilisearch-js", which is clearer I think.

File: resources/dependencies.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+- [`docs-searchbar.js`](https://github.com/meilisearch/docs-searchbar.js)
+- [`firestore-meilisearch`](https://github.com/meilisearch/firestore-meilisearch/)
+- [`gatsby-plugin-meilsearch`](https://github.com/meilisearch/gatsby-plugin-meilisearch/)
```
Comment (mdubus) [inline resources/dependencies.md:10]:
```suggestion
- [`gatsby-plugin-meilisearch`](https://github.com/meilisearch/gatsby-plugin-meilisearch/)
```

File: resources/dependencies.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+- [`instant-meilisearch`](https://github.com/meilisearch/instant-meilisearch/)
+- [`strapi-plugin-meilsearch`](https://github.com/meilisearch/strapi-plugin-meilisearch/)
```
Comment (mdubus) [inline resources/dependencies.md:12]:
```suggestion
- [`strapi-plugin-meilisearch`](https://github.com/meilisearch/strapi-plugin-meilisearch/)
```

File: resources/dependencies.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+## PHP SDK
+
+They depend on [`meilisearch-php`](https://github.com/meilisearch/meilisearch-php):
+- [`laravel/scout`](https://github.com/laravel/scout)
+- [`meiliseach-symfony`](https://github.com/meilisearch/meilisearch-symfony/)
```
Comment (mdubus) [inline resources/dependencies.md:18]:
```suggestion
- [`meilisearch-symfony`](https://github.com/meilisearch/meilisearch-symfony/)
```

File: resources/dependencies.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+## Python SDK
+
+They depend on [`meilisearch-python`](https://github.com/meilisearch/meilisearch-python):
+- [`docs-scraper`](https://github.com/meilisearch/docs-scraper/)
+
+## Ruby SDK
+
+They depend on [`meilisearch-ruby`](https://github.com/meilisearch/meilisearch-ruby):
+- [`meiliseach-rails`](https://github.com/meilisearch/meilisearch-rails/)
```
Comment (mdubus) [inline resources/dependencies.md:28]:
```suggestion
- [`meilisearch-rails`](https://github.com/meilisearch/meilisearch-rails/)
```

File: resources/dependencies.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+- The [code-sandbox of instant-meilisearch](https://codesandbox.io/s/ms-is-mese9?fontsize=14&hidenavigation=1&theme=dark)
+
+Also, even if they are not parts of the integration team, we should keep in mind the following projects strongly depend on `instant-meilisearch`:
+- the [`mini-dashboard`](https://github.com/meilisearch/mini-dashboard)
+- the demos of the DevRel team, like [the MoMA demo](https://github.com/meilisearch/demo-MoMA)
+
+## Docs-searchbar.js
+
+They depend on [`docs-searchbar.js`](https://github.com/meilisearch/docs-searchbar.js)
+- [`vuepress-plugin-meilsearch`](https://github.com/meilisearch/vuepress-plugin-meilisearch/)
```
Comment (mdubus) [inline resources/dependencies.md:48]:
```suggestion
- [`vuepress-plugin-meilisearch`](https://github.com/meilisearch/vuepress-plugin-meilisearch/)
```

File: resources/dependencies.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+
+## Cloud scripts
+
+They depend on [`cloud-scripts`](https://github.com/meilisearch/cloud-scripts)
+- [`meiliseach-aws`](https://github.com/meilisearch/meilisearch-aws/)
+- [`meiliseach-digitalocean`](https://github.com/meilisearch/meilisearch-gcp/)
+- [`meiliseach-gcp`](https://github.com/meilisearch/meilisearch-gcp/)
```
Comment (mdubus) [inline resources/dependencies.md:55]:
```suggestion
- [`meilisearch-aws`](https://github.com/meilisearch/meilisearch-aws/)
- [`meilisearch-digitalocean`](https://github.com/meilisearch/meilisearch-gcp/)
- [`meilisearch-gcp`](https://github.com/meilisearch/meilisearch-gcp/)
```

**Review by mdubus — CHANGES_REQUESTED** [review]

File: resources/dependencies.md [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+
+They depend on [`meilisearch-ruby`](https://github.com/meilisearch/meilisearch-ruby):
+- [`meiliseach-rails`](https://github.com/meilisearch/meilisearch-rails/)
+
+⚠️ Also, even if they are not parts of the integration team, we should keep in mind the following projects strongly depend on `meilisearch-ruby`:
+- the [`meilisearch-cloud`](https://github.com/meilisearch/meilisearch-cloud) backend (link only accessible internally)
+
+## Instant MeiliSearch
+
+They depend on [`instant-meilsearch`](https://github.com/meilisearch/instant-meilisearch/)
```
[the remark this answers is not in the document]
Comment (mdubus) [reply resources/dependencies.md:35]:
```suggestion
They depend on [`instant-meilisearch`](https://github.com/meilisearch/instant-meilisearch/)
```

**Review by mdubus — CHANGES_REQUESTED** [review]
Sorry I forgot this one on my first review 🙈

File: resources/dependencies.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (bidoubiwa) [reply resources/dependencies.md:7]: Ah yes sorry completely wrong of me

File: resources/dependencies.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bidoubiwa) [reply resources/dependencies.md:40]: Should I keep track of them myself meanwhile ?

**Review by bidoubiwa — DISMISSED** [review]
LGTM 🔥

File: resources/dependencies.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (curquiza) [reply resources/dependencies.md:35]:
```suggestion
They depend on [`instant-meilisearch`](https://github.com/meilisearch/instant-meilisearch/)
```

File: resources/dependencies.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (curquiza) [reply resources/dependencies.md:40]: Nope, not your job! This is the job of the devrel team to keep track of their own demos and be aware of their dependencies.

**Review by bidoubiwa — COMMENTED** [review]
LGTM 🔥🔥🔥

**Pull request merged**